### PR TITLE
Argus release 2.0.0 rc1

### PR DESCRIPTION
**Pull request by sagarmamdapure**
added pvc and endpoints in helm rbac

### CODE REVIEW

File: config-templates/argus/argus-config.yaml [hunk 1]
```diff
@@ -179,4 +179,6 @@ kube-state-metrics:
     - daemonsets
     - replicasets
     - statefulsets
-    - persistentvolumes
\ No newline at end of file
+    - persistentvolumes
+    - endpoint
```
Comment (vkumbhar94) [inline config-templates/argus/argus-config.yaml:183]: endpoints?

**Review by vkumbhar94 — COMMENTED** [review]
please incorporate suggested changes.

File: config-templates/argus/argus-config.yaml (hunk 1 above)
Comment (sagarmamdapure) [reply config-templates/argus/argus-config.yaml]: Thanks for pointing this out. Fixed

**Review by vkumbhar94 — APPROVED** [review]

**Comment by vkumbhar94** [comment]
Merging on behalf of Sagar.

**Pull request merged**